### ISSUE TITLE
securedrop-admin setup fails on Tails 4

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#4751]
## Description

We need to add a torify call to the `pip install` commands in `securedrop-admin setup`

## Steps to Reproduce

1. Install the Tails 4 beta: https://tails.boum.org/news/test_4.0-beta2/
2. Set up persistence
3. Clone the SecureDrop repo
4. From `~/Persistent/securedrop`, run `./securedrop-admin setup`

## Expected Behavior

apt and python dependencies are installed

## Actual Behavior

apt dependencies are installed, but then:

```
ERROR: Failed to install pip dependencies. Check network connection and try again.
```

## Comments

Rerunning with `-v` indicates that the problem is a connection failure. Prefixing the `./admin/.venv/bin/pip install -r admin/requirements.txt [...more options snipped....]` command with `torify` resolves. 

Related to #4669

**Comment by kushaldas** [comment]
@pradyunsg should we also file a bug against `pip` for this?

**Comment by pradyunsg** [comment]
If someone is willing to post the options passed to pip and more complete output from pip (redacting any sensitive information appropriately), it'll be much easier for me to tell if it's a pip thing or a tor/tails thing or an invocation issue. The arguments/output would help me confirm that.

---

I feel that this is more of a situation of needing to push pip's connections through Tor, when on Tails.

A quick look at the corresponding code makes me think that the issue is either in pip, or `admin/bootstrap.py:is_tails()` isn't working right. There's already `maybe_torify()` at that call site (I think the OP is suggesting that's what is needed). I'm not familiar with the code here, so I'm guessing here. :)

https://github.com/freedomofpress/securedrop/blob/7a9750d93b0d8be77fceca1a1bdb6446b5433038/admin/bootstrap.py#L185

**Comment by emkll** [comment]
Tails 4.0 is scheduled to be released on October 22 (per https://tails.boum.org/contribute/calendar/)

**Comment by redshiftzero** [comment]
yep looks like the issue is `is_tails()`, meaning this ticket has the same root cause as #4767

**Comment by zenmonkeykstop** [comment]
Filed a bug against Tails instead of pip: https://redmine.tails.boum.org/code/issues/17074

In the meantime, I'll add a check against /etc/os-release instead to see if it contains Tails-specific strings, which will unblock Tails 4.0 testing until the bug above is addressed.

**Comment by illumi420** [comment]
Python Dependencies problems (module Encodings failed + pip failure) for securedrop-admin is solved on Tails 5.4
